### ISSUE TITLE
Project Feedback

### ISSUE THREAD

**Issue by kellyskarritt** [kellyskarritt/MI-449-SS17-740-css-stealing-styles-xedvDK#1]
@chrisvfritz Can you take a look at this? It's [hosted here](https://kellyskarritt.github.io/MI-449-SS17-740-css-stealing-styles-xedvDK/) and meets the following criteria:

- [x] The page includes the name of your search engine in big letters (you can make it up), with a search `input` below it, then a button below that to run the search (the button doesn't have to do anything for this prototype)
- [x] The font size of the search engine name is at least 70 pixels, so that it's very easy to see
- [x] `span` elements are wrapped around each letter of the name, like this: `<span>G</span><span>o</span><span>o</span><span>g</span><span>l</span><span>e</span>`, so that they can be styled individually with classes
- [x] The on-hover colors from the paragraphs [on this page](http://skillful-clouds.surge.sh/) are used to color individual letters of your name (similar to the letters in Google's logo)
- [x] Inspect the `input` element at the bottom of the [_Prototyping states_ page of the lesson](https://msu.lansing.codes/courses/MI-000-SS00-000/lessons/css-stealing-styles/5) to create an identical-looking search box for your own site (it should even use the same styles on `focus`)
- [x] The styles from the "Create Account" button [on this page](http://plant-servant.surge.sh/) are used for your "Search" button (it should even use the same styles on `hover` and `active`)
- [x] There are no stylelint errors in your CSS (this will also be expected for all future projects)
- [x] All your CSS exists in a `.css` file

**Comment by KatieMFritz** [comment]
Hey Kelly! Chris asked me to cover this review for him.

Nice start on this. 🌈 ✨ Your page looks very clean and polished! However, there are some criteria you haven't met exactly yet:

### Use exact colors

> The on-hover colors from the paragraphs [on this page](http://skillful-clouds.surge.sh/) are used to color individual letters of your name (similar to the letters in Google's logo)

We want you to use the _exact_ colors used on that page - you'll have to use some fancy Inspect skills to get them! :art: 

* * *
### Replicate the `input` element from the lesson

> Inspect the `input` element at the bottom of the [Prototyping states page of the lesson](https://msu.lansing.codes/courses/MI-000-SS00-000/lessons/css-stealing-styles/5) to create an identical-looking search box for your own site (it should even use the same styles on `focus`)

You're close, but when I focus on your `input` element, the border turns blue for me. The `input` in the lesson gets a dark green border on focus. Check it out again and find the exact color (and any other styles) that gets applied on :focus. 

* * *
### Exactly replicate button

> The styles from the "Create Account" button [on this page](http://plant-servant.surge.sh/) are used for your "Search" button (it should even use the same styles on `hover` and `active`)

Your button is really close! However, there are some differences. I think you did the exact same thing I did when I was testing this project over the weekend - I made a `button` element, but the Stripe page actually uses different elements (`ul`, `li`, `a`). You don't _have_ to use the same elements that they do, but it might make it easier. 

Some things that are visually different that I noticed: the blue text seems to be different, the mouse pointer doesn't have the same behavior when hovering or clicking, and the `active` state is different.

Let me know if you get stuck on this - I'll be on Slack. 😄 

* * *

### Tip: Naming classes

Something to think about when you're making up class names for CSS is how you might be able to reuse that class in the future. For example, your letters have class names that are tied to those specific letters. But what if you want to have two orange letters? You'd have to give the second one a class of `A`, even if it's not "A." You ran into this already with `A` and `A2` classes. 

What about using the color names instead? 😄 

* * *
After you’ve made your changes and pushed them to GitHub and your hosted site, give it a once-over to make sure it looks right, then comment back here and I’ll take another look.

Thanks! :rocket:

**Comment by kellyskarritt** [comment]
I made the changes! Thanks!

**Comment by KatieMFritz** [comment]
Hi Kelly, getting closer! 🍪 🌴 
### Name
For the colored letters, we want them to be different colors _all_ the time, not just on hover - like the Google logo. 

I see you've added the colors from the example page and changed the class names. 👍 
### Input
The input looks good! ✔️ 
 ### Button
The button looks the same to me normally and on `hover`, but different on `active`. See these images:
![kelly](https://cloud.githubusercontent.com/assets/7085184/22897677/a266a2aa-f1f2-11e6-9d9d-3c03344126f0.png) Yours
![demo](https://cloud.githubusercontent.com/assets/7085184/22897681/a5acacde-f1f2-11e6-9ae0-f49e19d1a5b3.png) Demo

And a small CSS challenge for you: Right now, you're using `br` to get your button to be on its own line. There are ways to do that with CSS instead. Hint - play around with the `display` property.

**Comment by kellyskarritt** [comment]
Done! Thanks

**Comment by KatieMFritz** [comment]
That was fast! 🚀 

### Name
Now I'm seeing rainbow colors for the name, and the project-specific colors when I hover. I want to see each letter be a different color, like it is now, but I want those colors to be the project colors (that you stole from the page with the hover colors - you're currently using those colors only on hover). Does that make sense?

### Button
Looks great! I see you removed the `br` too. 👍

**Comment by kellyskarritt** [comment]
Oh! So we don't need a hover? I just need the exact colors? I thought the hover had to be the exact colors from the project

**Comment by KatieMFritz** [comment]
Right, we want you to get the _hover_ colors from the source page, but use them to color the _normal_ state for your letters on your page. The reason is so you could practice using the `:hov` selector tool in Chrome dev tools to copy hover state styles. But we want your site name to look more like Google's logo.

>The on-hover colors from the paragraphs on this page are used to color individual letters of your name (similar to the letters in Google's logo)

**Comment by kellyskarritt** [comment]
Thank you for clearing that up! Sorry I was super confused. It's done (I hope)!

**Comment by KatieMFritz** [comment]
It's no problem - you aren't the only person who was confused by that. 🙂 Sometimes we write instructions that seem clear to us, but they turn out to be more ambiguous!

Your site looks wonderful. :shipit:
